pages/docs/getting-started/index.md

```diff
@@ -3,19 +3,99 @@ title: Introduction
 date: 2019-02-16T13:56:52+01:00
 weight: 0
 categories:
-- getting-started
+- introduction
 aliases:
 - '/v1/guide/'
 - '/v1/guide/index.html'
 ---
 
-AsyncAPI is an open source initiative that seeks to improve the current state of Event-Driven Architectures (EDA). Our long-term goal is to make working with EDAs as easy as it is to work with REST APIs. That goes from documentation to code generation, from discovery to event management. Most of the processes you apply to your REST APIs nowadays would be applicable to your event-driven/asynchronous APIs too.
+Welcome to the AysncAPI documentation section where you get to learn about solutions to automate and formalize the documentation or code generation of your event-driven (micro)services.
```

Comment:
So this introduction needs to be an introduction to the tutorial, not the spec :) 

Let's update this section to explain detail what this tutorial is going to teach us and what we are going to build.